src/backend/prisma/schema.prisma

```diff
@@ -10,29 +7,122 @@ generator client {
   provider = "prisma-client-js"
 }
 
-model Post {
-  id        Int      @id @default(autoincrement())
-  createdAt DateTime @default(now())
-  updatedAt DateTime @updatedAt
-  title     String
-  content   String?
-  published Boolean  @default(false)
-  author    User     @relation(fields: [authorId], references: [id])
-  authorId  Int
+enum CR_Type {
+  DESIGN_ISSUE
+  NEW_FUNCTION
+  OTHER
+  STAGE_GATING
+  WP_INITIATION
```

Comment:
I know you were saying something about having different tables for these (like Change Requests Table and then some other table), so I can add those or just keep it as is for now